### ISSUE TITLE
Big Missiles don't have sound

### ISSUE THREAD

**Issue by MavroGada2** [TheSuperHackers/GeneralsGamePatch#1775]
Scud Storm Missile don't have sound. Their silence is noticed as they fall from the sky with zero sound.

I added ``` SoundMoveLoop ``` with ``` AudioEvent BattleShipIncomingWhistle ``` and changed some parameters. 

Sounds could be better but overall better than muted version.

### Original:

https://user-images.githubusercontent.com/103841664/226191534-34431aaf-fdd3-4001-803c-23be5fa0884e.mp4

### With Sound:

https://user-images.githubusercontent.com/103841664/226191623-30c29b7d-2209-40d0-98bf-f018ba4c6215.mp4

**Comment by xezon** [comment]
Good idea. What about the Nuke Missile? And Scud Launcher missile? I think if we do this, then there should be consistency.

**Comment by MavroGada2** [comment]
There are no sounds for any missile iirc. I am trying to add but stuck at nuke missile for now.

**Comment by xezon** [comment]
Note that Nuke Cannon and Inferno Cannon are supposed to have projectile whistle, but it is broken in Zero Hour for some reason.

* #1596

**Comment by MavroGada2** [comment]
Scud Launcher and tomahawk missile needs more work on them but nuke missile is ready. We can get better results if we add different sounds to game.

https://user-images.githubusercontent.com/103841664/226289503-56bb53cf-09e1-4e60-8d85-acfae032fcda.mp4


https://user-images.githubusercontent.com/103841664/226289771-545a0219-b4b3-4346-ae63-0e3e0f960af8.mp4

**Comment by MavroGada2** [comment]
For scud laucher and tomahawk ```Delay``` is  working terrible so I added custom empty sound file.

They all look good expect for tomahawk. Laucher and Storm have same sound with different tone(sound is from CCG USA01 scud cinematic most likely). Changed Nuke Missile tone.


<details>
  <summary>Tomahawk and Scud Launcher:</summary>


https://user-images.githubusercontent.com/103841664/226727695-f733ceca-ee0e-42d9-9065-c8fdddda1e61.mp4

</details>

<details>
  <summary>Scud Storm:</summary>


https://user-images.githubusercontent.com/103841664/226728427-6270a93f-1215-4535-82bf-2a5364fe0ebb.mp4

</details>

<details>
  <summary>Nuke Missile:</summary>

https://user-images.githubusercontent.com/103841664/226728951-6d310a22-d85b-400d-b632-1a7f39cf3b96.mp4

</details>

**Comment by xezon** [comment]
Nuke missile sound appears to work well.

Tomahawk missile sound is not good.

Scud Launcher missile is alright.

Scud Storm missiles sound lags too much, meaning sounds play while missiles are already destroyed.
